config.yaml

```diff
@@ -44,25 +44,25 @@ oracle:
     api_key: BZM7P395BQS1YFKQMA2AK2ACQKWGEW4JB3
     apis:
       gas_price: https://api.etherscan.io/api?module=gastracker&action=gasoracle&apikey=
+    gas_price_domainIDs: [ 0 ]
```

Comment:
Don't think this resolves the problem if we want to register another domain with the same implementation, but a different api-key. This way I don't see how the gas_price_domainIDs would ever have 2 elements.